community-content/pytorch_pre_built_images_deployment/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -0,0 +1,4106 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    IS_GOOGLE_CLOUD_NOTEBOOK = os.path.exists("/opt/deeplearning/metadata/env_version")
</pre><p>would it be ok to always us --user? (I think it might be)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
It would be ok. But I would like to leave it as is to match the other notebooks.